reference/providers/oracle.md

```diff
@@ -1,8 +1,90 @@
 ---
 layout: single
-title:  "Oracle"
+title:  "Oracle Cloud"
 sidebar:
   nav: reference
 ---
 
 {% include toc %}
+
+For more information about Oracle Cloud or any of the terms used below, please consult
+[Oracle Cloud Infrastructure Documentation](https://docs.cloud.oracle.com/iaas/Content/home.htm).
+
+## Resource mapping
+
+### Account
+The Oracle Cloud provider in Spinnaker maps a Spinnaker [__Account__](/concepts/providers/#accounts)
+to an [Oracle Cloud Infrastructure user]( https://cloud.oracle.com/en_US/tryit) in 
```

Comment:
@shihchang 
Instead of linking to /tryit/, can you link to something more specifically about `user`? [This](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/addingusers.htm#one) maybe?

Reply:
That link looks better, changed.